packages/core/components/AutoField/index.tsx

```diff
@@ -1,5 +1,6 @@
 import getClassNameFactory from "../../lib/get-class-name-factory";
```

Comment:
This implementation is currently causing a React re-mount whenever the input data is changed when used inside a custom field. Need to investigate further.